### DISCUSSION TITLE
Do we even have support?

### DISCUSSION THREAD

**Discussion by coolacpc** [community/community#56491] [GitHub Education]
### Hi everyone,

Been trying to get student pack for 2 months now, and i always get an error.
i am currently doing my degree online in a university in another country. I have Provided every single document, original and translated, but still getting rejected!

I tried contacting support around 16 days ago, but still response.
Thank you!

![Screenshot 2023-05-26 173807](https://github.com/community/community/assets/13906653/ddc97910-3049-4c14-a288-a94a7e621cc4)

**Comment by mrepol742** [comment]
They said the document you provided does not indicate `virtual learning` since you are remote. Also according to them your document indicates its an acceptance letter which is not allowed only academic enrollment is acceptable. additionally you may provide dated school id, since you are remote your document must state that you are not near to any of it's campuses.

**Reply by coolacpc** [reply]
The thing is i provided proof of enrollment and not acceptance letter.

**Comment by samus-aran** [comment]
Hi @coolacpc 👋 

Lets work through these rejection reasons:

1. For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status. You've stated you are outside the country, then we would like to see evidence that your course is online or remote. 
2. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps. If text isn't legible and perhaps mistakes the evidence as an enrolment letter.
3. It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it.
4. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Work through these points and let us know how you get on 👍
